REQUIRE

```diff
@@ -1,4 +1,3 @@
-julia 0.5
-TikzPictures 0.3.5
-LightGraphs 0.6.0
-Compat 0.8.0
+julia 0.7-beta2
```

Comment:
Do we want this, or just `julia 0.7-`?

Reply:
Pretty sure I saw this being recommended.
Once julia 0.7 proper is out we can upgrade to that.